### PR TITLE
fix(webhooks): added info about attachment checks #10045

### PR DESCRIPTION
**Pull request by thekeogh**
ref: https://github.com/screencloud/apps/issues/10045

image

### CODE REVIEW

**Review by gerardwilson1 — APPROVED** [review]

File: docs/content/webhooks-app.md [hunk 1]
```diff
@@ -289,6 +289,29 @@ Every error response you get from a webhook `POST` will be in the format outline
 }
 ```
 
+## Problematic Attachments
+
+When you include `attachments` with a specified `contentType` or `image`, we'll verify not just the URL's validity and existence, but also confirm that the content type matches an image. Should this verification fail, you'll be alerted that the attachment has been excluded from the post. The webhook post will proceed successfully; however, the invalid images will be omitted.
```
Comment (shannen-smyth) [inline docs/content/webhooks-app.md:294]: Should this be:
```
 specified `contentType` of `image
```

Comment (thekeogh) [reply docs/content/webhooks-app.md:294]: Ahh, it should indeed be that

File: docs/src/components/sidebar/treeNode.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ const TreeNode = ({ className = '', setCollapsed, collapsed, url, title, items,
   return (
     <li className={calculatedClassName}>
       {/* TODO: Remove the below url check, its just to hide the item for now */}
```
Comment (shannen-smyth) [inline docs/src/components/sidebar/treeNode.js:28]: We could remove this comment now too?

Comment (thekeogh) [reply docs/src/components/sidebar/treeNode.js:28]: Ahh I did this, put it back cos there was an error then forgot to remove it, what a wally. Will remove now

**Review by shannen-smyth — APPROVED** [review]

**Pull request merged**